mutiny-wasm/src/lib.rs

```diff
@@ -362,6 +362,16 @@ impl NodeManager {
         Ok(self.inner.get_bitcoin_price().await?)
     }
 
+    pub async fn export_json(&self) -> Result<JsValue, MutinyJsError> {
```

Comment:
maybe this should just be a string? I think this will deserialize to a javascript object